system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -230,6 +239,9 @@ match_types! {
 		MultiLocation { parents: 1, interior: Here } |
 		MultiLocation { parents: 1, interior: X1(_) }
 	};
+	pub type TreasuryPallet: impl Contains<MultiLocation> = {
+		MultiLocation { parents: 1, interior: X1(PalletInstance(kusama_runtime_constants::TREASURY_PALLET_ID)) }
+	};
```

Comment:
```suggestion
```

not needed, and also `TreasuryPallet` could be kind of misleading it is relay or local treasury?
 replaced bellow with `Equals<RelayTreasuryLocation>`